tours/tcga/tour.txt

```diff
@@ -1,23 +1,37 @@
-#featAssayPlot1_DataBoxOpen;<p align="left">To visualize the differential expression, we used the <i>Data parameters</i> box to:</p><ul><li>assign <i>ERBB2</i> to the Y-axis</li><li>show gene expression values from the assay that holds log<sub>2</sub> count per million (CPM) values</li><li>separate cancer types on the X axis</li></ul>
+#featAssayPlot1;<p align="left">Rahman <i>et al.</i> examined the effects of <i>HER2</i> (ERBB2) overexpression in breast cancer cells.</p><p>Here, we note that a subset of breast cancer tumor sample display markedly higher expression of <i>HER2</i>.</p>
+#featAssayPlot1_DataBoxOpen;<p align="left">To visualize the differential expression, we used the <font color="#402ee8">Data parameters</font> box to:</p><ul><li>assign <i>HER2</i> (ERBB2) to the Y-axis</li><li>show gene expression values from the assay that holds log<sub>2</sub> count per million (CPM) values</li><li>separate tumor and control samples on the X axis</li></ul>
 #featAssayPlot1_Downsample;<p align="left">For this panel too, we apply downsampling.</p>
-#collapse_featAssayPlot1_SelectBoxOpen;<p align="left">To highlight the breast cancer tumor samples from female patients selected in the <font color="#402ee8">Column data plot</font> panel, we:<ul><li>Select it as the source of point selection</li><li>apply the selection with a color effect (red, by default)</li></ul>
-#featAssayPlot1;<p align="left">This applies the selected color to the selected samples, leaving others in their default color.</p><p align="left">The resulting effect confirms that the highest levels of <i>ERBB2</i> are seen in breast cancer cells of female patients, supporting the overexpression reported by <a href="https://academic.oup.com/bioinformatics/article/31/22/3666/240143" target="_blank">Rahman et al. (Bioinformatics, 2015)</a>.</p>
+#collapse_featAssayPlot1_SelectBoxOpen;<p align="left">To restrict the plot to breast cancer tumor samples selected in the <font color="#402ee8">Column data plot</font> panel:<ul><li>we select it as the source of point selection</li><li>we apply the selection with a restriction effect</li></ul>
+#featAssayPlot1;<p align="left">This displays only the selected breast cancer tumor and control samples, effectively removing others from this plot.</p><p align="left">The resulting effect emphasises a subset of <i>HER2</i>-positive breast cancer tumor samples.
+#heatMapPlot1;<p align="left"><font color="#402ee8">Heat map</font> panels offer a convenient way to jointly visualise sample- and feature-level information.</p><p align="left">Here, we visualise the top 60 <i>HER2</i> signature genes, selected by largest absolute coefficients reported in Supplementary Table S4 of Rahman <i>et al.</i>, and identified from <a href="http://bioconductor.org/packages/Rsubread/" target="_blank">Rsubread</a> FPKM values.</p>
```

Comment:
We should probably have italics for packages - probably something to mention in the wiki.